Code.gs

```diff
@@ -21,14 +21,28 @@
 *=========================================
 */
 
-var sourceCalendars = [                // The ics/ical urls that you want to get events from along with their target calendars (list a new row for each mapping of ICS url to Google Calendar)
-                                       // For instance: ["https://p24-calendars.icloud.com/holidays/us_en.ics", "US Holidays"]
-                                       // Or with colors following mapping https://developers.google.com/apps-script/reference/calendar/event-color,
-                                       // for instance: ["https://p24-calendars.icloud.com/holidays/us_en.ics", "US Holidays", "11"]
-  ["icsUrl1", "targetCalendar1"],
-  ["icsUrl2", "targetCalendar2"],
-  ["icsUrl3", "targetCalendar1"]
-
+var sourceCalendars = [
+
+  // The ics/ical urls that you want to get events from along with their target calendars (list a new row for each mapping of ICS url to Google Calendar)
+  // For instance: ["https://p24-calendars.icloud.com/holidays/us_en.ics", "US Holidays"]
+
+  // Or with additional Options:
+  // - colorId: Colors of the Events with the following mapping https://developers.google.com/apps-script/reference/calendar/event-color
+  // - calendarName: Alternative name for the parent calendar
+  // for instance:
+  // ["https://p24-calendars.icloud.com/holidays/us_en.ics", "US Holidays", {
+  //   colorId: "1",
+  //   calendarName: 'Parent Calendar Name'
+  // }]
+
+  ["icsUrl1", "targetCalendar1", {
+    colorId: "1",
+    calendarName: 'Parent Calendar Name 1'
+  }],
+  ["icsUrl2", "targetCalendar2", {
+    calendarName: 'Parent Calendar Name 2'
```

Comment:
Let's use proper ical property names here:
colorId -> color
calendarName -> name

[https://www.rfc-editor.org/rfc/rfc5545](https://www.rfc-editor.org/rfc/rfc5545)
[https://www.rfc-editor.org/rfc/rfc7986](https://www.rfc-editor.org/rfc/rfc7986)

Reply:
Also need to change 2x **parentCal** to **name** at line 460++